groupactivity.md

```diff
@@ -0,0 +1,4 @@
+# Group Activity
+
+Team members:
+* [Nathan Whitney](https://github.com/the29ster)
```

Comment:
Add a space before a list

Also, include all developer team members in the list